client/my-sites/post-type-filter/index.jsx

```diff
@@ -3,44 +3,49 @@
  */
 import React, { PropTypes } from 'react';
 import { connect } from 'react-redux';
-import { find, includes, reduce } from 'lodash';
+import { compact, find, includes, reduce } from 'lodash';
 
 /**
  * Internal dependencies
  */
+import { areAllSitesSingleUser } from 'state/selectors';
 import { getSelectedSiteId } from 'state/ui/selectors';
-import { getSiteSlug, isJetpackSite } from 'state/sites/selectors';
-import { getNormalizedPostCounts } from 'state/posts/counts/selectors';
+import { isJetpackSite, isSingleUserSite, getSiteSlug } from 'state/sites/selectors';
+import { getNormalizedMyPostCounts, getNormalizedPostCounts } from 'state/posts/counts/selectors';
 import { mapPostStatus } from 'lib/route/path';
 import UrlSearch from 'lib/mixins/url-search';
 import QueryPostCounts from 'components/data/query-post-counts';
 import SectionNav from 'components/section-nav';
 import NavTabs from 'components/section-nav/tabs';
 import NavItem from 'components/section-nav/item';
 import Search from 'components/search';
+import AuthorSegmented from './author-segmented';
 
 const PostTypeFilter = React.createClass( {
 	mixins: [ UrlSearch ],
 
 	propTypes: {
+		authorToggleHidden: PropTypes.bool,
 		siteId: PropTypes.number,
-		query: PropTypes.object,
+		query: PropTypes.shape( {
+			author: PropTypes.number, // User ID
+			status: PropTypes.string,
+			type: PropTypes.string.isRequired,
+		} ).isRequired,
 		jetpack: PropTypes.bool,
 		siteSlug: PropTypes.string,
 		counts: PropTypes.object
 	},
 
-	getDefaultProps() {
-		return {
-			query: {}
-		};
-	},
-
 	getNavItems() {
-		const { query, siteSlug, jetpack, counts } = this.props;
+		const { query, siteId, siteSlug, jetpack, counts } = this.props;
 
 		return reduce( counts, ( memo, count, status ) => {
-			if ( ! jetpack && ! count && ! includes( [ 'publish', 'draft' ], status ) ) {
+			// * Always add 'publish' and 'draft' tabs
```

Comment:
This isn't true for an Atomic site in production:

![](https://cldup.com/KP3MBlAJ8w.png)

but this PR fixes it:

![](https://cldup.com/mUD2Ws1kK1.png)

🎉